<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConvertDialog</class>
 <widget class="QDialog" name="ConvertDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Direção-Geral do Território - Recart - Converter Cartografia</string>
  </property>
  <property name="locale">
   <locale language="Portuguese" country="Portugal"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="1" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Registo</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit"/>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="maximum">
            <number>0</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Entrada</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="5" column="0" colspan="8">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Opções de conversão</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="0" colspan="3">
             <widget class="QCheckBox" name="forceGeomCheckBox">
              <property name="text">
               <string>Forçar dimensões geometria</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QCheckBox" name="overwriteLNCheckBox">
              <property name="text">
               <string>Mapear com nome camada</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QComboBox" name="fieldsComboBox"/>
            </item>
            <item row="6" column="0">
             <widget class="QCheckBox" name="cmapCheckBox">
              <property name="text">
               <string>Mapeamento personalizado</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Campo Mcod</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QCheckBox" name="caliasCheckBox">
              <property name="text">
               <string>Alias p / Códigos</string>
              </property>
             </widget>
            </item>
            <item row="7" column="2">
             <widget class="QgsFileWidget" name="aliasQgsFileWidget">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="storageMode">
               <enum>QgsFileWidget::GetFile</enum>
              </property>
             </widget>
            </item>
            <item row="6" column="2">
             <widget class="QgsFileWidget" name="mapQgsFileWidget">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="storageMode">
               <enum>QgsFileWidget::GetDirectory</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="forcePolygonCheckBox">
              <property name="text">
               <string>Forçar poligonos</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QCheckBox" name="forceCloseCheckBox">
              <property name="text">
               <string>Forçar fecho geometria</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QCheckBox" name="cellHeaderOrigin">
              <property name="text">
               <string>Aproveitar apenas a origem das células CAD</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="1" colspan="3">
          <widget class="QComboBox" name="connCombo"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Destino</string>
           </property>
          </widget>
         </item>
         <item row="3" column="6">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Versão</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="5">
          <widget class="QgsProjectionSelectionWidget" name="mQgsProjectionSelectionWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="7">
          <widget class="QgsFileWidget" name="inputQgsFileWidget">
           <property name="storageMode">
            <enum>QgsFileWidget::GetMultipleFiles</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="8">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="5">
          <widget class="QLineEdit" name="lineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Coordenadas</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Pasta temporária</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Ficheiro(s)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="7">
          <widget class="QgsFileWidget" name="configQgsFileWidget">
           <property name="storageMode">
            <enum>QgsFileWidget::GetDirectory</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Esquema</string>
           </property>
          </widget>
         </item>
         <item row="2" column="6">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>NdD</string>
           </property>
          </widget>
         </item>
         <item row="2" column="7">
          <widget class="QComboBox" name="nddCombo"/>
         </item>
         <item row="3" column="7">
          <widget class="QComboBox" name="vrsCombo"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
  <customwidget>
   <class>QgsProjectionSelectionWidget</class>
   <extends>QWidget</extends>
   <header>qgsprojectionselectionwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConvertDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inputQgsFileWidget</sender>
   <signal>fileChanged(QString)</signal>
   <receiver>ConvertDialog</receiver>
   <slot>changeInputFile(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmapCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>ConvertDialog</receiver>
   <slot>changeCustomMap(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>caliasCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>ConvertDialog</receiver>
   <slot>changeAliasMap(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>connCombo</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>ConvertDialog</receiver>
   <slot>changeConn(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>589</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>overwriteLNCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>fieldsComboBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>528</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeInputFile(QString)</slot>
  <slot>changeCustomMap(int)</slot>
  <slot>changeAliasMap(int)</slot>
  <slot>changeConn(QString)</slot>
 </slots>
</ui>
